i18n/locales/en.ts

```diff
@@ -61,7 +61,7 @@ const translations = {
       title: 'Enjoy!',
       contributions_by: 'Contributions by',
       many_more:
-        'and <a href="https://github.com/saving-satoshi/saving-satoshi/graphs/contributors" target="_blank" rel="noreferrer">many more</a>.',
+        ', <href="https://twitter.com/VEINTIUNow" target="_blank">@VEINTIUNow</a> and <a href="https://github.com/saving-satoshi/saving-satoshi/graphs/contributors" target="_blank" rel="noreferrer">many more</a>.',
```

Comment:
Please add your social to the following code in `/app/[lang]/about/page.tsx` instead.

```ts
const CONTRIBUTORS = [
  { username: 'adamcjonas', url: 'https://twitter.com/adamcjonas' },
  { username: 'afterburn', url: 'https://github.com/afterburn' },
  { username: 'benalleng', url: 'https://github.com/benalleng' },
  { username: 'ecurrencyhodler', url: 'https://twitter.com/ecurrencyhodler' },
  { username: 'gbks', url: 'https://twitter.com/gbks' },
  { username: 'satsie', url: 'https://twitter.com/satsie' },
  { username: 'shaavan', url: 'https://github.com/shaavan' },
  { username: 'vinayaksh42', url: 'https://github.com/vinayaksh42' },
]
```